doc/sources/guide/packaging-osx.rst

```diff
@@ -108,54 +108,65 @@ Complete guide
     $ pip install git+https://github.com/pyinstaller/pyinstaller.git@develop
 
 
+#. Export the ``HOOKSPATH`` environment variable::
+
+    $ export HOOKSPATH=`python -c "
+    import imp, os
+    print(os.path.join(imp.find_module('kivy')[1], 'tools/packaging/pyinstaller_hooks'))"`
+
 #. Package your app using the path to your main.py::
 
-    $ pyinstaller -y --clean --windowed --name touchtracer /usr/local/share/kivy-examples/demo/touchtracer/main.py
+    $ pyinstaller -y --clean --windowed --name touchtracer \
+      --additional-hooks-dir $HOOKSPATH \
+      --runtime-hook $HOOKSPATH/rt-hook-kivy.py \
```

Comment:
+1 for passing the hook in the command, this method will make the most sense once the kivy hook gets merged in pyinstaller.